crates/uv-workspace/src/pyproject.rs

```diff
@@ -113,6 +113,20 @@ impl AsRef<[u8]> for PyProjectToml {
     }
 }
 
+/// A specifier item in a PEP 735 Dependency Group
+#[derive(Serialize, Deserialize, Debug, Clone, PartialEq, Eq)]
+#[serde(rename_all = "kebab-case", untagged)]
+pub enum DependencyGroupSpecifier {
+    /// PEP 508 requirement string
+    Requirement(String),
+    /// Include another dependency group
+    #[serde(rename_all = "kebab-case")]
```

Comment:
I think it's enough for this to be included at the top-level, or is it not?

Reply:
This is derived from the implementation at https://github.com/PyO3/pyproject-toml-rs/pull/24 per https://github.com/astral-sh/uv/pull/8104#discussion_r1800654513 — I don't have any preference, personally.